### ISSUE TITLE
Deprecation of {view} in Ember 1.13

### ISSUE THREAD

**Issue by benedikt-d** [ef4/liquid-fire#308]
Using `{{view}}` or any path based on it (application/templates/components/liquid-modal.hbs' @ L4:C6) has been deprecated. [deprecation id: view.keyword.view]

See http://emberjs.com/deprecations/v1.x#toc_view-and-controller-template-keywords for more details.

**Comment by ef4** [comment]
Yes, this is the final deprecation I'm working on before I ship liquid-fire-2.0-beta.0.
